### PR TITLE
Update DG with link/unlink command implementation

### PR DESCRIPTION
**Pull request by xyT-T**

### CODE REVIEW

File: docs/DeveloperGuide.md
Comment (rionshocker) [inline docs/DeveloperGuide.md:1]: Remember to add the use cases for unlink, delete, mark and unmark task

**Review by rionshocker — CHANGES_REQUESTED** [review]
LGTM! Except for a nit to fix.

File: docs/DeveloperGuide.md
Comment (xyT-T) [reply docs/DeveloperGuide.md:1]: thanks for reminding! will add that in now.

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -702,6 +792,90 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+**Extensions**
+
+* 1a. Any of the mandatory fields not specified.
+
+    * 1a1. JobFestGo informs user that mandatory fields not specified.
+
+      Use case ends.
+
+* 1b. Task does not already exist.
```
Comment (rionshocker) [inline docs/DeveloperGuide.md:867]:
```suggestion
* 1b. Task does not exist.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -702,6 +792,90 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    * 1c1. JobFestGo informs user that task does not already exist.
```
Comment (rionshocker) [inline docs/DeveloperGuide.md:869]:
```suggestion
    * 1c1. JobFestGo informs user that task does not exist.
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -702,6 +792,90 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+**Extensions**
+
+* 1a. Any of the mandatory fields not specified.
+
+    * 1a1. JobFestGo informs user that mandatory fields not specified.
+
+      Use case ends.
+
+* 1b. Task does not already exist.
```
Comment (rionshocker) [inline docs/DeveloperGuide.md:837]: Same thing as below

**Review by rionshocker — CHANGES_REQUESTED** [review]
LGTM! Just some minor changes to make

**Review by rionshocker — APPROVED** [review]
LGTM!

**Pull request merged**